app/src/main/java/com/cmput301w23t40/capturetheqr/LibraryActivity.java

```diff
@@ -46,13 +59,17 @@ protected void onCreate(Bundle savedInstanceState) {
         qrCodeDataList = new ArrayList<>();
         String visFake = new String("vis\nfake\nlist\n");
 
-        QRCode qr1 = new QRCode("fakeHash", "name1", visFake, 10);
+        QRCode qr1 = new QRCode("fakeHash", "BFG5DGW54", visFake, 10);
         QRCode qr2 = new QRCode("fakeHash", "name2", visFake, 20);
         qrCodeDataList.addAll(Arrays.asList(qr1, qr2));
         qrCodeList = new QRCodeList(this, qrCodeDataList);
 
         qrCodeView.setAdapter(qrCodeList);
 
+        DB.setDB(FirebaseFirestore.getInstance());
```

Comment:
maybe we could move this line to MainActivity.create(), since it is always the first method to run. It would be better if we have DB setup initially rather than setting it up when we first need it

Reply:
Done